lib/Voxa-Opearlo.js

```diff
@@ -56,7 +54,6 @@ function register(skill, config) {
       alexaEvent.intent.name,
```

Comment:
seems to me that if we're recording errors then the event name should be 'Error' and not the intent name. And then we should have something like "message": error.message as the fourth parameter